modules/ipi-install-configuring-managed-secure-boot-in-the-install-config-file.adoc

```diff
@@ -0,0 +1,38 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
+// installing/installing_bare_metal_ipi/ipi-install-installations-workflow.adoc
+[id="configuring-managed-secure-boot-in-the-install-config-file_{context}"]
+= Configuring managed Secure Boot in the `install-config.yaml` file (optional)
+
+You can enable managed Secure Boot when deploying an installer-provisioned cluster using Redfish BMC addressing, such as `redfish`, `redfish-virtualmedia`, or `idrac-virtualmedia`. To enable managed Secure Boot, add the `bootMode` configuration setting to each node:
+
+[source,yaml]
+.Example
+----
+hosts:
+  - name: openshift-master-0
+    role: master
+    bmc:
+      address: redfish://<out_of_band_ip> <1>
+      username: <user>
+      password: <password>
+    bootMACAddress: <NIC1_mac_address>
+    rootDeviceHints:
+     deviceName: "/dev/sda"
+    bootMode: UEFISecureBoot <2>
+----
+
+<1> Ensure the `bmc.address` setting uses `redfish`, `redfish-virtualmedia`, or `idrac-virtualmedia` as the protocol. See "BMC addressing for HPE iLO" or "BMC addressing for Dell iDRAC" for additional details.
```

Comment:
IBM SG. Change the second sentence to have the prepositional phrase start the sentence: "For additional information, see..."